projects/globiancedex/index.js

```diff
@@ -0,0 +1,18 @@
+const retry = require("../helper/retry");
+const axios = require("axios");
+
+async function fetch() {
+  const response = (
+    await retry(
+      async (bail) => await axios.post("https://dexapi.globiance.com/get-stats")
+    )
+  ).data;
+
+  const tvl = response.data.tvl;
+
+  return tvl || 0;
```

Comment:
here it's better to avoid `|| 0` otherwise if the script fails we'll write a 0 incorrectly into our database

failing the script is better @waynebruce0x